src/qml/WelcomeScreen.qml

```diff
@@ -327,24 +327,36 @@ Page {
           Text {
             Layout.margins: 6
             Layout.maximumWidth: collectionView.width - 12
-            text: qsTr("To improve stability for everyone, QField collects and sends anonymized metrics. To disable, click on the button below.")
+            text: qsTr("To improve stability for everyone, QField collects and sends anonymized metrics.")
             font: Theme.defaultFont
             horizontalAlignment: Text.AlignHCenter
             wrapMode: Text.WordWrap
           }
 
           RowLayout {
             spacing: 6
-            Layout.alignment: Qt.AlignVCenter | Qt.AlignHCenter
+            Layout.alignment: Qt.AlignVCenter
             Layout.bottomMargin: 10
             QfToolButton {
-              iconSource: Theme.getThemeIcon('ic_close_white_24dp')
+              text: qsTr('Thank you!')
```

Comment:
Not sure we can have text in tool buttons, let's see :)

You could use QfButton there, that might require a tiny bit more tweaking.